etc/testing/circle/install.sh

```diff
@@ -54,9 +54,8 @@ fi
 
 # Install helm
 if [ ! -f cached-deps/helm ]; then
-  HELM_VERSION=3.5.4
+  HELM_VERSION=3.14.0
```

Comment:
this might not be the only place. 😬 [couple places where its messy](https://github.com/pachyderm/pachyderm/blob/43e7589b958e078b8b4d53288780e0876f891197/.circleci/main.yml#L283) the helm ci test & build use `3.5.2`

Reply:
Good call.  I got a bunch more outdated helms, minikubes, and kinds in circleci/main.yml.